source/conf.py

```diff
@@ -34,16 +34,22 @@
         "rapids_conda_packages": f"rapids={stable_version} python=3.12 cuda-version=12.5",
         "rapids_pip_index": "https://pypi.nvidia.com",
         "rapids_pip_version": stable_version,
+        # SageMaker Notebook Instance examples need to stay pinned to an older RAPIDS until this is resolved:
+        # https://github.com/rapidsai/deployment/issues/520
+        "rapids_sagemaker_conda_packages": "rapids=24.12 python=3.12 cuda-version=12.5",
     },
     "nightly": {
-        "rapids_version": f"{nightly_version}-nightly",
+        "rapids_version": f"{nightly_version}",
```

Comment:
Nothing in the repo relies on this `-nightly` being there, as far as I can tell.

```shell
git grep rapids_version
```

And in fact, this difference is causing some bugs. Look at https://docs.rapids.ai/deployment/nightly/platforms/databricks/#install-rapids-and-dask .... it's saying you should install `dask-cuda==25.02-nightly`, which does not exist.

<img width="773" alt="Screenshot 2025-02-11 at 9 00 42 PM" src="https://github.com/user-attachments/assets/e9d665a5-6e04-45d8-82d4-efd4d2f7c69d" />

Reply:
Yeah I think this might be a holdover from a previous naming scheme. Fine to remove it.